build.py

```diff
@@ -436,9 +436,20 @@ def run(self):
             "'closure-library-read-only' to 'closure-library'.\n"
             "Please rename this directory.")
     else:
-      print("""Error: Closure not found.  Read this:
+    	try:
+        calcdeps = import_path(os.path.join(
+        os.path.pardir, "google-closure-library", "closure", "bin", "calcdeps.py"))
+  		except ImportError:
+    		if os.path.isdir(os.path.join(os.path.pardir, "closure-library-read-only")):
```

Comment:
For this specific implementation, this check and the printed error are unnecessary, no?